docs/TROUBLE.md

```diff
@@ -0,0 +1,162 @@
+
+## 7. Confirm MAVLink message flow from ground station to flight controller
+
+Using your ground station connected to your Tx module, verify that you can change flight modes (Using the ground station, not the radio) and successfully download parameters.
+
+- If you can't change flight modes or download parameters or issue other commands via the ground station, check your wiring to ensure that the correct serial tx output on your ground control device or wireless module or serial adapter is connected to the correct serial rx input on your Tx module.
+
+- If flight mode changes initiated by the ground control system are correctly reported back to the ground control system and FM sensor, but parameter download fails, see [Confirm reliable message delivery](TROUBLE.md#Confirm reliable message delivery) below.
+
+## 8. Confirm RC control is passed from the radio to the flight controller
+
+Use the radio calibration page of your ground station (connected either via your Tx module or directly to USB) to verify all RC controls are working as expected.
+
+- If you use RC/_CHANNELS\_OVERRIDE instead of a CRSF Rx output and are testing with QGroundControl, switch to Mission Planner or APM Planner 2.0 since QGroundControl doesn't show RC\_CHANNELS\_OVERRIDE on this page.
+
+- If the display doesn't follow stick movement and you use a CRSF connection between your Rx and your flight controller, check that you have connected the wire to the correct pins/pads.
+
+- For CRSF, check that you have correctly [configured ArduPilot](ARDUPILOT.md#crsf-receiver) to use this input.
+
+- For CRSF, check that you have correctly [configured your Rx](CRSF.md#mlrs-rx-module-setup) for crsf output.
+
+- If your Rx [doesn't provide an RC output](ELRS_RECEIVERS.md#connections) or you don't want to use it, check that you have set [Rx Snd RcChannel](PARAMETERS.md#rx-snd-rcchannel) to rc override.
+
+## 9. Confirm reliable message delivery
+
+Use your ground control station connected to your Tx module to examine the message loss rate both during parameter download and for a few minutes after it completes.
+
+For QGroundControl, select Application Settings -> MAVLink and scroll down to MAVLink Link Status.
+
+For Mission Planner, select status in the upper right, move the windows so you can see both and then refresh parameters from the full parameters page.
+
+You should see very few or no messages lost.
+
+- If you see more than the occasional lost message or if parameter download takes more time than expected (8 to 60 seconds, depending on mode and ArduPilot version), check that [rx-ser-link-mode](PARAMETERS.md#rx-ser-link-mode) is set to "mavlink" or "mavlinkX" and that [rx-snd-radiostat](PARAMETERS.md#rx-snd-radiostat) is set to "ardu_1" or "meth_b"
+
+- Check your [ArduPilot setup](CRSF.md#ardupilot-setup) to be sure that the baud rate between the Rx and  and your configured Stream Rates are appropriate for your radio mode.
```

Comment:
stream rates shouls be lower case

this statement is highly missleading as it implies these would have to be set carefully for mLRS to work properly, which is NOT the case !!! In fact, we spend lots of time to make this be not true. 
so, at most it should come accros like a suggestion but not a "be sure"

Reply:
But on old versions of Ardupilot before I fixed some of the flow control issues, these settings may well make a difference.  But I'll reword.

Reply:
what actually did cross my mind is that we maybe should say, here or somewhere, that there is no need to fool with the baud rate setting. I got a bit an impression that the first thing many users do is to change the baudrates ...

maybe one shoudl have a comment in the above on the ArduPilot version. With a brief recap of implications, and the advice to go to latest ArduPilot.

Reply:
I think I will also do a quick start guide for the same core use case covered here.  I'll probably mention that it's best not to fiddle with the baud rates there.